src/atoms/LinkBarButtonBase.js

```diff
@@ -1,7 +1,5 @@
-import {
-	BlockLink,
-	LinkBarLinkBase,
-} from '..';
+import {	BlockLink	} from '..';
+import {	LinkBarLinkBase	} from '..';
```

Comment:
Why didn't eslint catch this? :-/

Reply:
Fixed, but eslint did not care. as long as it's one space. it do not complaint.